docs/source/usersguide/geometry.rst

```diff
@@ -0,0 +1,353 @@
+    +----------------------+------------------------------+---------------------------+
+    | Infinite cylinder    | :math:`(y-y_0)^2 + (z-z_0)^2 | :class:`openmc.XCylinder` |
+    | parallel to          | - R^2 = 0`                   |                           |
+    | :math:`x`-axis       |                              |                           |
+    +----------------------+------------------------------+---------------------------+
+    | Infinite cylinder    | :math:`(x-x_0)^2 + (z-z_0)^2 | :class:`openmc.YCylinder` |
+    | parallel to          | - R^2 = 0`                   |                           |
+    | :math:`y`-axis       |                              |                           |
+    +----------------------+------------------------------+---------------------------+
+    | Infinite cylinder    | :math:`(x-x_0)^2 + (y-y_0)^2 | :class:`openmc.ZCylinder` |
+    | parallel to          | - R^2 = 0`                   |                           |
+    | :math:`z`-axis       |                              |                           |
+    +----------------------+------------------------------+---------------------------+
+    | Sphere               | :math:`(x-x_0)^2 + (y-y_0)^2 | :class:`openmc.Sphere`    |
+    |                      | + (z-z_0)^2 - R^2 = 0`       |                           |
+    +----------------------+------------------------------+---------------------------+
+    | Cone parallel to the | :math:`(y-y_0)^2 + (z-z_0)^2 | :class:`openmc.XCone`     |
+    | :math:`x`-axis       | - R^2(x-x_0)^2 = 0`          |                           |
+    +----------------------+------------------------------+---------------------------+
+    | Cone parallel to the | :math:`(x-x_0)^2 + (z-z_0)^2 | :class:`openmc.YCone`     |
+    | :math:`y`-axis       | - R^2(y-y_0)^2 = 0`          |                           |
+    +----------------------+------------------------------+---------------------------+
+    | Cone parallel to the | :math:`(x-x_0)^2 + (y-y_0)^2 | :class:`openmc.ZCone`     |
+    | :math:`z`-axis       | - R^2(z-z_0)^2 = 0`          |                           |
+    +----------------------+------------------------------+---------------------------+
+    | General quadric      | :math:`Ax^2 + By^2 + Cz^2 +  |  :class:`openmc.Quadric`  |
+    | surface              | Dxy + Eyz + Fxz \\+Gx + Hy + |                           |
+    |                      | Jz + K = 0`                  |                           |
+    +----------------------+------------------------------+---------------------------+
+
+Each surface is characterized by several parameters. As one example, the
+parameters for a sphere are the :math:`x,y,z` coordinates of the center of the
+sphere and the radius of the sphere. All of these parameters can be set either
+as optional keyword arguments to the class constructor or via attributes::
+
+  sphere = openmc.Sphere(R=10.0)
```

Comment:
Yeah, that occurred to me too. Not sure if it's worth fixing and breaking any code already using it. It really should be lower-case.